### PR TITLE
fix: clean example of jsonResponse usage in webhooks

### PR DESCRIPTION
**Pull request by p-desaintchamas**

### CODE REVIEW

File: recipes/webhooks_example.ipynb [hunk 1]
```diff
@@ -438,21 +508,35 @@
     "If you used & hosted the base webhook provided, the webhook should:\n",
     "\n",
     " - Create an issue with information that three bboxes were found, instead of one\n",
+    " - Create an issue with info that the first bbox is too large\n",
     " - Send the asset back to the labeling queue (status `ONGOING`)\n",
     "\n",
-    "If you haven't just yet, you can still visit the address here : "
+    "If you haven't deploed your webhook just yet, you can still visit the address here : "
```
Comment (hguerni) [inline recipes/webhooks_example.ipynb:514]: deployed*

File: recipes/webhooks_example.ipynb [hunk 2]
```diff
@@ -189,52 +189,110 @@
...
+    "from kili.plugins import PluginCore\n",
+    "\n",
+    "\n",
     "def check_rules_on_label(label: Dict):\n",
-    "    #custom methods \n",
-    "    print('Custom method - checking number of bboxes')\n",
+    "    \"\"\"\n",
+    "    Custom function to check rules on label.\n",
+    "    For basic rules, a handy object is `search` that \\\n",
+    "        provide various analytics on objects\n",
```
Comment (gheorghetutunaru) [inline recipes/webhooks_example.ipynb:204]:
```suggestion
    "        provides various analytics on objects\n",
```

File: recipes/webhooks_example.ipynb [hunk 3]
```diff
@@ -189,52 +189,110 @@
...
     "class PluginHandler(PluginCore):\n",
     "    \"\"\"\n",
     "    Custom plugin instance\n",
     "    \"\"\"\n",
     "\n",
+    "    def check_complex_rules_on_label(self, asset_id: str):\n",
+    "        \"\"\"\n",
+    "        Custom method to check if a box is larger than 33% of the image\n",
+    "        For basic rules, a handy object is `search` that \\\n",
+    "            provide various analytics on objects\n",
```
Comment (gheorghetutunaru) [inline recipes/webhooks_example.ipynb:240]:
```suggestion
    "            provides various analytics on objects\n",
```

File: recipes/webhooks_example.ipynb [hunk 4]
```diff
@@ -189,52 +189,110 @@
...
     "\n",
-    "                self.kili.append_to_issues(\n",
-    "                    label_id=label['id'],\n",
+    "                self.kili.create_issues(\n",
+    "                    label_id_array=[label['id']],\n",
     "                    project_id=project_id,\n",
-    "                    text=issues_array[i],\n",
+    "                    text_array=[issue['text']],\n",
+    "                    object_mid_array=[issue['mid']]\n",
     "                )\n",
```
Comment (gheorghetutunaru) [inline recipes/webhooks_example.ipynb:287]: Why not create all the issues at once in batch ?

Comment (p-desaintchamas) [reply recipes/webhooks_example.ipynb:287]: Not possible, misconception, the batch only works for one issue per asset

Comment (gheorghetutunaru) [reply recipes/webhooks_example.ipynb:287]: Hmm, okay

**Review by gheorghetutunaru — APPROVED** [review]

**Review by hguerni — APPROVED** [review]

**Pull request merged**